WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/usecases/PublishPostImmediatelyUseCase.kt

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.ui.posts.prepublishing.home.usecases
+
+import org.wordpress.android.fluxc.model.PostModel
+import org.wordpress.android.fluxc.model.post.PostStatus
+import org.wordpress.android.fluxc.model.post.PostStatus.SCHEDULED
+import org.wordpress.android.ui.posts.EditPostRepository
+import org.wordpress.android.util.DateTimeUtilsWrapper
+import javax.inject.Inject
+
+class PublishPostImmediatelyUseCase @Inject constructor(private val dateTimeUtilsWrapper: DateTimeUtilsWrapper) {
+    fun updatePostToPublishImmediately(
+        editPostRepository: EditPostRepository,
+        publishPost: Boolean
```

Comment:
It seems this parameter is not used anymore - to be more precise it's always "true". 

As discussed above, I think the parameter is confusing anyway `updatePostToPublishImmediately` with parameter `publishPost` -> it shouldn't do anything at all if the parameter is false imo.

Reply:
Yes, this is so true. Thanks for spotting. I moved it here when doing the refactoring but it's definitely not needed. 🙏